dotcom-rendering/src/components/Branding.importable.tsx

```diff
@@ -84,14 +85,14 @@ const brandingAboutLink = (palette: Palette, format: ArticleFormat) => {
 			return [
 				invariantStyles,
 				css`
-					color: ${palette.text.branding};
+					color: ${palette('--branding-link-text')};
 					${until.desktop} {
-						color: ${palette.text.standfirst};
+						color: ${palette('--standfirst-text')};
 					}
 					a {
-						color: ${palette.text.branding};
+						color: ${palette('--branding-link-text')};
 						${until.desktop} {
-							color: ${palette.text.standfirst};
+							color: ${palette('--standfirst-text')};
 						}
 					}
```

Comment:
By using `color: inherit;` we can remove the loose coupling of media queries.

I would also encourage these to be mobile-first, using `from.*` rather than `until.*`, but agreed this is an existing pattern that can be addressed outside this PR.